### PR TITLE
Update 1.0.0

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,67 +1,61 @@
...
     requirements:
       build:
         - {{ compiler('cxx') }}
         - cmake
-        - ninja-base
+        - ninja
+        - python  # [win]
       host:
-        - libsolv >=0.7.19
+        - libsolv 0.7.22
```
Comment (jezdez) [inline recipe/meta.yaml:46]: Should this be pinned to a specific version?

Comment (katietz) [reply recipe/meta.yaml:46]: yes, as we want to define here a minimal version.  a >= would pick always a max version, which is not necessarily what you wanted here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,67 +1,64 @@
...
+  sha256: f77b858012d3a93b75565c60ab7594c9508bed559b8369aca32f736e9186a733
   patches:
-    # Fix some compilation issues.
-    - patches/libmamba_CMakeLists.txt.patch
-    - patches/libmamba_src_core_context.cpp.patch
+    - add_find_package_fmt.patch
 
 build:
-  number: 0
+  number: 2
```
Comment (jezdez) [inline recipe/meta.yaml]: Should we reset that to 0?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,67 +1,64 @@
...
     version: {{ libmamba_version }}
-    script: build_libmamba.sh                     # [unix]
-    script: bld_libmamba.bat                      # [win]
+    script: build_mamba.sh                       # [unix]
+    script: build_mamba.bat                      # [win]
     build:
       run_exports:
         - {{ pin_subpackage('libmamba', max_pin='x.x') }}
-      missing_dso_whitelist:  # [s390x]
-        - $RPATH/ld64.so.1    # [s390x] Known s390x `ld64.so` issue.
```
Comment (jezdez) [inline recipe/meta.yaml]: I'm wondering if this was legitimately fixed? I think this was added by Anaconda staff in the past.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -81,36 +78,31 @@ outputs:
...
-        - $RPATH/ld64.so.1    # [s390x] Know s390x `ld64.so` issue.
       ignore_run_exports:
-        - spdlog-fmt-embed    # Only using headers
+        - spdlog
     requirements:
       build:
         - {{ compiler('cxx') }}
         - cmake
-        - ninja-base
+        - ninja
```
Comment (jezdez) [inline recipe/meta.yaml:95]: I don't understand the difference, I don't see a file size difference

File: recipe/meta.yaml [hunk 5]
```diff
@@ -81,36 +78,31 @@ outputs:
...
       host:
         - python
         - pip
-        - setuptools
-        - wheel
```
Comment (jezdez) [inline recipe/meta.yaml]: Do we need those for building the Python package?

**Review by jezdez — COMMENTED** [review]
Just a few comments @katietz, thank you for your work!

**Comment by jezdez** [comment]
@jaimergp Would you mind taking a look at this?

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,67 +1,66 @@
...
+        - reproc-cpp 14.2.4
+        - spdlog
         - yaml-cpp
         - termcolor-cpp
         - cli11
+        - fmt
+        - libiconv                               # [osx]
         - winreg                                 # [win]
+      run:
+        - libsolv >=0.7.19
```
Comment (jaimergp) [inline recipe/meta.yaml]: Shouldn't this be >=0.7.22 (as in host)?

```suggestion
        - libsolv >=0.7.22
```

**Comment by jezdez** [comment]
Should we rename the repo to mamba-feedstock to denote change in strategy?

**Comment by jezdez** [comment]
@katietz Is there anything left on your side for this?

**Comment by katietz** [comment]
no, fixes have a PR on CF too, otherwise I just wait for your approval on it

**Review by jezdez — APPROVED** [review]

**Comment by katietz** [comment]
Merge it with just one approver, but actual there was a second-one involved, so it is fine

**Pull request merged**